### PR TITLE
Expand improvements

### PR DESCRIPTION
**Pull request by tedivm**
* Don’t expand immediately- wait at least 10k ticks for scouting.

* Don’t claim rooms with only 1 source until empire is more established (five cities).

* Continue supporting new rooms even after they become “self sufficient”

### CODE REVIEW

File: src/extends/room/territory.js [hunk 1]
```diff
@@ -223,7 +230,7 @@ Room.getCityScore = function (roomName) {
     return false
   }
 
-  // As written the INTEL_SOUrCES check should cover this, hoever these were added after so for the current running user
+  // As written the INTEL_SOURCES check should cover this, hoever these were added after so for the current running user
```
Comment (TheRealMaxion) [inline src/extends/room/territory.js]: typo

File: src/extends/room/territory.js [hunk 2]
```diff
@@ -223,7 +230,7 @@ Room.getCityScore = function (roomName) {
...
   // this check is required until the intel tills in. This check should be safe to remove a week or so after this code
```
Comment (TheRealMaxion) [inline src/extends/room/territory.js]: has it been a week yet?

File: src/extends/room/territory.js (hunk 1 above)
Comment (tedivm) [reply src/extends/room/territory.js]: This PR fixed the typo

**Comment by TheRealMaxion** [comment]
Would that part be needed again after a respawn or deployment on a new server?

**Comment by tedivm** [comment]
No- it was a check for when the intel existed but wasn't complete. On anything installed after that change (which should be everything since we also respawned) the intel will either not be there at all or it'll be complete.

**Comment by TheRealMaxion** [comment]
Okay, good. :)

**Pull request merged**